instrumentation/otel-webserver-module/src/nginx/ngx_http_opentelemetry_module.c

```diff
@@ -1101,18 +1101,18 @@ static void stopMonitoringRequest(ngx_http_request_t* r,
     }
 
     APPD_SDK_STATUS_CODE res;
-    unsigned int errCode=0;
```

Comment:
Same as above..Lets use a better practice

Reply:
This is unsigned int initialised to zero value. Do you think we really need to define this else where